src/main/java/com/Elessar/app/util/kafka/ProducerCreator.java

```diff
@@ -13,7 +13,10 @@
     public static <K, V> Producer<K, V> create(Serializer<K> keySerializer, Serializer<V> valueSerializer) {
         Properties props = new Properties();
         props.put(ProducerConfig.BOOTSTRAP_SERVERS_CONFIG, "localhost:9092");
-        props.put(ProducerConfig.CLIENT_ID_CONFIG, "localhost:9092");
+        props.put(ProducerConfig.CLIENT_ID_CONFIG, "http://127.0.0.1:9092");
```

Comment:
...
ok, let's add a TODO or FIXME for the IP part here. we should really use something like "ip:9000".
Please fix the port though, we should use 9000 (MyServer's port) here!